smpplib/client.py

```diff
@@ -64,13 +75,13 @@ class Client(object):
     _socket = None
     sequence_generator = None
 
-    def __init__(self, host, port, timeout=5, sequence_generator=None):
+    def __init__(self, host, port, timeout=5, sequence_generator=None, start_sequence=None):
         self.host = host
         self.port = int(port)
         self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self._socket.settimeout(timeout)
         if sequence_generator is None:
-            sequence_generator = SimpleSequenceGenerator()
+            sequence_generator = SimpleSequenceGenerator(start_sequence=start_sequence)
```

Comment:
If someone has their own custom sequence generator implementation this implementation is likely to break after your patch applied as probably there will be no arguments in its  `__init__`  and you always pass one from now on.